docs/en/query_language/syntax.md

```diff
@@ -26,23 +26,33 @@ SQL-style and C-style comments are supported.
 SQL-style comments: from `--` to the end of the line. The space after `--` can be omitted.
 Comments in C-style: from `/*` to `*/`. These comments can be multiline. Spaces are not required here, either.
 
-## Keywords
+## Keywords {#syntax-keywords}
 
-Keywords (such as `SELECT`) are not case-sensitive. Everything else (column names, functions, and so on), in contrast to standard SQL, is case-sensitive. Keywords are not reserved (they are just parsed as keywords in the corresponding context).
+Keywords (such as `SELECT`) are not case-sensitive. Everything else (column names, functions, and so on), in contrast to standard SQL, is case-sensitive.
 
-## Identifiers
+Keywords are not reserved (they are just parsed as keywords in the corresponding context). If you use [identifiers](#syntax-identifiers) the same as the keywords, enclose them into quotes, for example, `"column_name"`. The query `SELECT "FROM" FROM table_name` is valid if the table `table_name` has column with the name `"FROM"`.
```

Comment:
I'd drop `for example, `"column_name"`` because the actual example is the following sentence.